.github/workflows/release.yml

```
@@ -77,21 +78,36 @@ jobs:
 
       - name: Packing Linux builds
         run: |
-          pushd publish_linux
-          tar -czvf ../release_output/ryujinx-${{ steps.version_info.outputs.build_version }}-linux_x64.tar.gz publish
-          popd
+         #Sadly windows does not support libfuse!
+         #sudo wget -O appimage-builder-x86_64.AppImage https://github.com/AppImageCrafters/appimage-builder/releases/download/v1.1.0/appimage-builder-1.1.0-x86_64.AppImage
+         #sudo chmod +x appimage-builder-x86_64.AppImage
+         #sudo mv appimage-builder-x86_64.AppImage /usr/bin/appimage-builder
+         
+         #appimage-builder --recipe AppImageBuilder.yml --skip-test
+         mv /home/runner/work/Ryujinx/Ryujinx/Ryujinx-*-x86_64.AppImage publish
+         
+         # pushd publish_linux
+         # tar -czvf ../release_output/ryujinx-${{ steps.version_info.outputs.build_version }}-linux_x64.tar.gz publish
+         # popd
 
+         # Most users won't need this and these are mainly for debug so not packaging as an appimage
           pushd publish_linux_sdl2_headless
           tar -czvf ../release_output/sdl2-ryujinx-headless-${{ steps.version_info.outputs.build_version }}-linux_x64.tar.gz publish
           popd
 
           pushd publish_linux_ava
           tar -czvf ../release_output/test-ava-ryujinx-${{ steps.version_info.outputs.build_version }}-linux_x64.tar.gz publish
           popd
+          cd ./publish
         shell: bash
+      
+      - uses: AppImageCrafters/build-appimage@master
+        with:
+          recipe: "../AppImageBuilder.yml"
+          args: "--skip-test"
```

We have a reduced amount of allowed actions on the org.

I'm not a fan of 1) docker based actions 2) the fact it refer to the master branch.

Please prefer on host installation.


```suggestion
      - name: Packing AppImage (Linux)
        run: |
         sudo pip3 install appimage-builder==1.0.0
         appimage-builder --recipe AppImageBuilder.yml --skip-test
        shell: bash
```

---

Yeah I didn't want to use this either. Unfortunately appimages cannot be built in windows or WSL.